pyEpiabm/pyEpiabm/sweep/initial_place_sweep.py

```diff
@@ -0,0 +1,127 @@
+#
+# Sweep to initialise people present in a place
+#
+
+from pyEpiabm.core import Parameters
+
+from .abstract_sweep import AbstractSweep
+from .update_place_sweep import UpdatePlaceSweep
+
+
+class InitialisePlaceSweep(AbstractSweep):
+    """Class to update people in the "Place"
+    class.
+    """
+    def __call__(self):
+        """Given a population structure, updates the people
+        present in each place at a specific timepoint. The
+        explicit code handles the fixed population which are
+        not changed later on in the simulation. To initialise
+        the variable population (for example OutdoorSpace only
+        has a variable population) one instance of
+        UpdatePlaceSweep is called at the end to instantiate that.
+
+        Parameters
+        ----------
+        time : float
+            Current simulation time
+
+        """
+
+        # Double loop over the whole population, clearing places
+        # of the variable population and refilling them.
+
+        # May want a param dict to be called in from file etc.
+        # place_params = Parameters.instance().place_params
```

Comment:
Do we still need this line of commented code?

Reply:
nope